bin/test.sh

```diff
@@ -111,6 +115,10 @@ if $ios82_only || $all_tests; then
     run_ios_tests "8.2" "ios82" "@skip-ios82|@skip-ios81|@skip-ios8|@skip-ios-all|@skip-ios7up"
 fi
 
+if $ios83_only || $all_tests; then
+    run_ios_tests "8.3" "ios83" "@skip-ios83|@skip-ios82|@skip-ios81|@skip-ios8|@skip-ios-all|@skip-ios7up"
```

Comment:
looks good to me, though we probably want to run some of these tests just to see if something actually got fixed!